image_provisioner/playbooks/setup-image.yaml

```diff
@@ -2,6 +2,5 @@
 - name: setup base image
   user: root
   hosts: localhost
-
   roles:
-    - setup-image
+    - { role: setup-image, when: docker_storage_driver == "devicemapper" }
```

Comment:
@jeremyeder @sjug Is there some reason we don't want to enable overlay for qcow2?   I can see where we might want to do some work to allow everything to be on /dev/vda1, but it should work OK with the second device like we are doing for AMIs.